lumen/ai/prompts/VegaLiteAgent/main.jinja2

```diff
@@ -6,5 +6,5 @@ Generate the plot the user requested as a vega-lite specification.
 - The data will be provided separately. Never provide a data field.
 - Ensure you use the column names verbatim.
 - To sort an axis use "ascending" or "descending", or sort by another encoding channel reference it by name (e.g. `sort: y` or `sort: -y` for descending order).
-- Integer columns like year (e.g., 2010, 2011) should be treated as ordinal, not numerical or temporal, but '2010-01' or '2011-01-01' should be treated as temporal.
+- Integer columns like year (e.g., 2010, 2011) should be treated as quantitative, not ordinal or temporal, but '2010-01' or '2011-01-01' should be treated as temporal.
```

Comment:
Not sure why I put ordinal and not numerical for integers.

Reply:
Because that's correct, years should be ordinal.

Reply:
Please revert.

Reply:
Quite context dependent I guess for a line chart numerical is right, for a bar chart I much prefer the ordinal behavior.

Reply:
Okay, let's leave it.